### PR TITLE
Hawaii Standard Deduction

### LINKED ISSUE #2101: Hawaii standard deduction

### PR DESCRIPTION
**Pull request by ruarua7788**
Fixes #2101

### CODE REVIEW

File: policyengine_us/parameters/gov/states/hi/tax/income/deductions/standard/amount.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+description: Hawaii provides filers a standard deduction of this amount, depending on their filing status.
+metadata:
+  label: Hawaii standard deduction
+  period: year
+  unit: currency-USD
+  breakdown:
+  - filing_status
+  reference:
+  - title: 2022 Hawaii Tax Form Instruction Booklet p20
+    href: https://files.hawaii.gov/tax/forms/2022/n11ins.pdf.
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/hi/tax/income/deductions/standard/amount.yaml:10]:
```suggestion
    href: https://files.hawaii.gov/tax/forms/2022/n11ins.pdf#page=20
```

File: changelog_entry.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+- bump: minor
+  changes:
+    added:
+      - Hawaii Standard Deduction
```
Comment (MaxGhenis) [inline changelog_entry.yaml:4]:
```suggestion
      - Hawaii standard deduction.
```

**Review by MaxGhenis — APPROVED** [review]

File: policyengine_us/parameters/gov/states/hi/tax/income/deductions/standard/amount.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+description: Hawaii provides filers a standard deduction of this amount, depending on their filing status.
+metadata:
+  label: Hawaii standard deduction
+  period: year
+  unit: currency-USD
+  breakdown:
+  - filing_status
+  reference:
+  - title: 2022 Hawaii Tax Form Instruction Booklet p20
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/hi/tax/income/deductions/standard/amount.yaml:9]:
```suggestion
  - title: 2022 Hawaii Tax Form Instruction Booklet line 23
```

**Comment by ruarua7788** [comment]
Hello Max. Thank you for your suggestions. I have already applied them.

File: policyengine_us/parameters/gov/states/hi/tax/income/deductions/standard/amount.yaml (hunk 3 above)
Comment (ruarua7788) [reply policyengine_us/parameters/gov/states/hi/tax/income/deductions/standard/amount.yaml]: Hello Max, thank you for your suggestions. I have already applied these changes and pushed a new commit.

**Comment by MaxGhenis** [comment]
Thanks @ruarua7788! FYI I've added `Fixes #2101` to the description, which links this PR to the issue so it closes it upon merging, which I'm doing now!

**Comment by ruarua7788** [comment]
Thanks @MaxGhenis !

**Pull request merged**